Doc/using/cmdline.rst

```diff
@@ -1100,3 +1107,13 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. envvar:: PYTHON_PRESITE=package.module
```

Comment:
Sphinx is really stupid :-( With your syntax, it cannot create a reference from the -X option doc to this variable, it warns:

```
'envvar' reference target not found: PYTHON_PRESITE
```

I suggest:

```suggestion
.. envvar:: PYTHON_PRESITE
```

Reply:
If we want to change this back, we should also change `PYTHONDUMPREFSFILE=FILENAME
` above. I noticed and worked around this problem in references I made for Blurb and in cmdline.rst above.

Reply:
Yeah, please change also PYTHONDUMPREFSFILE. There is a reference to this env var:

```
Misc/NEWS.d/3.11.0a1.rst:A debug variable :envvar:`PYTHONDUMPREFSFILE` is added for creating a dump
```

I don't know why Sphinx cannot handle this case.

Reply:
Changed in the upcoming push.